deps/ReactantExtra/tblgen/mlir-jl-tblgen.cc

```diff
@@ -61,7 +64,8 @@ int main(int argc, char **argv) {
   }
   disableModuleWrap = disableModuleWrapOpt;
 
-  return TableGenMain(argv[0], [](raw_ostream& os, const RecordKeeper &records) {
-    return generator(records, os);
-  });
+  return TableGenMain(argv[0],
+                      [](raw_ostream &os, const RecordKeeper &records) {
+                        return generator(records, os);
+                      });
 }
```

Comment:
I'm surprised it doesn't add a missing newline here:
```suggestion
}

```